media-gfx/sane_backends/sane_backends-1.0.29.recipe

```diff
@@ -78,10 +84,7 @@ BUILD_PREREQUIRES="
 
 BUILD()
 {
-	libtoolize --force --copy --install
-	aclocal
-	autoconf
-	automake --add-missing
+	export BACKENDS="abaton agfafocus apple artec artec_eplus48u as6e avision bh canon canon630u canon_dr cardscan coolscan coolscan2 coolscan3 dc25 dc210 dc240 dell1600n_net dmc epjitsu epson epson2 epsonds fujitsu genesys gt68xx hp hp3500 hp3900 hp4200 hp5400 hp5590 hpljm1005 hs2p ibm kodak kodakaio kvs1025 kvs20xx leo lexmark ma1509 magicolor matsushita microtek microtek2 mustek mustek_usb nec net niash pie pieusb pixma plustek plustek_pp ricoh ricoh2 rts8891 s9036 sceptre sharp sm3600 sm3840 sp15c st400 stv680 tamarack teco1 teco2 teco3 test u12 umax umax_pp umax1220u xerox_mfp p5 pnm"
```

Comment:
What's the point of this?

Reply:
Some backends are specific to certain OS platforms (like Mac OS X) or don't build/test properly. This helps filter them out.

Reply:
Doesn't the configure script detect that automatically?

Reply:
Basically, I filtered out drivers that didn't build/test properly on Haiku. Very minor issue, but helps
isolates/filter out faulty (or unwanted) drivers under test review.

Reply:
I don't think it's worth it, we should just leave everything enabled.

Reply:
Won't work...some filters are not 100%. We need to filter them out.

Reply:
Can you please be more specific? Just having unit tests fail is not a reason to disable them.

Reply:
Fixed!!